client/lib/signup/hint-data.js

```diff
@@ -1,673 +1,677 @@
+	'Fashion design': i18n.translate( 'Fashion design' ),
+	Feelings: i18n.translate( 'Feelings' ),
+	Festival: i18n.translate( 'Festival' ),
+	'Fiction writing': i18n.translate( 'Fiction writing' ),
+	'Film production': i18n.translate( 'Film production' ),
+	Films: i18n.translate( 'Films' ),
+	Financial: i18n.translate( 'Financial' ),
+	'Financial consultant': i18n.translate( 'Financial consultant' ),
+	'Financial services': i18n.translate( 'Financial services' ),
+	'Fine art': i18n.translate( 'Fine art' ),
+	'First nations': i18n.translate( 'First nations' ),
+	Fishing: i18n.translate( 'Fishing' ),
+	Fitness: i18n.translate( 'Fitness' ),
+	Florist: i18n.translate( 'Florist' ),
+	'Flying club': i18n.translate( 'Flying club' ),
+	Food: i18n.translate( 'Food' ),
+	'Food critic': i18n.translate( 'Food critic' ),
+	Football: i18n.translate( 'Football' ),
+	'Freelance designer': i18n.translate( 'Freelance designer' ),
+	'Freelance developer': i18n.translate( 'Freelance developer' ),
+	'Freelance marketer': i18n.translate( 'Freelance marketer' ),
+	'Freelance services': i18n.translate( 'Freelance services' ),
+	'Freelance writer': i18n.translate( 'Freelance writer' ),
+	'Funeral services': i18n.translate( 'Funeral services' ),
+	Furniture: i18n.translate( 'Furniture' ),
+	Games: i18n.translate( 'Games' ),
+	Gaming: i18n.translate( 'Gaming' ),
+	Garments: i18n.translate( 'Garments' ),
+	Genealogy: i18n.translate( 'Genealogy' ),
+	Glamour: i18n.translate( 'Glamour' ),
+	Gloves: i18n.translate( 'Gloves' ),
+	God: i18n.translate( 'God' ),
+	Golf: i18n.translate( 'Golf' ),
+	Gospel: i18n.translate( 'Gospel' ),
+	'Government and politics': i18n.translate( 'Government and politics' ),
+	'Graphic design': i18n.translate( 'Graphic design' ),
+	'Graphic novels': i18n.translate( 'Graphic novels' ),
+	Graphics: i18n.translate( 'Graphics' ),
+	'Green technology': i18n.translate( 'Green technology' ),
+	Groceries: i18n.translate( 'Groceries' ),
+	'Guitar teacher': i18n.translate( 'Guitar teacher' ),
+	Guns: i18n.translate( 'Guns' ),
+	Gym: i18n.translate( 'Gym' ),
+	HIV: i18n.translate( 'HIV' ),
+	HVAC: i18n.translate( 'HVAC' ),
+	Hacking: i18n.translate( 'Hacking' ),
+	'Hair salon': i18n.translate( 'Hair salon' ),
+	'Hair styles': i18n.translate( 'Hair styles' ),
+	Happiness: i18n.translate( 'Happiness' ),
+	Hardware: i18n.translate( 'Hardware' ),
+	Healing: i18n.translate( 'Healing' ),
+	Health: i18n.translate( 'Health' ),
+	'Health and beauty': i18n.translate( 'Health and beauty' ),
+	'Health and fitness': i18n.translate( 'Health and fitness' ),
+	Healthcare: i18n.translate( 'Healthcare' ),
+	'Heat transfer vinyl': i18n.translate( 'Heat transfer vinyl' ),
+	Hinduism: i18n.translate( 'Hinduism' ),
+	History: i18n.translate( 'History' ),
+	Hobbies: i18n.translate( 'Hobbies' ),
+	Holiday: i18n.translate( 'Holiday' ),
+	'Holiday rentals': i18n.translate( 'Holiday rentals' ),
+	Holidays: i18n.translate( 'Holidays' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **97** times:
`translate( 'Holiday' )` *ES Score: 17*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/translate-suggestions-about)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).